src/frontend/src/handler/create_table.rs

```diff
@@ -250,7 +250,7 @@ pub(crate) fn gen_materialized_source_plan(
             required_cols,
             out_names,
         )
-        .gen_create_mv_plan(source.name.clone())?
+        .gen_create_mv_plan(source.name.clone(), "".into())?
```

Comment:
Not sure whether `""` or a sql null is better in these cases (incl. `gen_create_index_plan` below).

Reply:
🤔, I'm not sure. Let's keep it for now.